front/src/router/index.js

```diff
@@ -1,14 +1,39 @@
 import Vue from 'vue';
 import VueRouter from 'vue-router';
-import Home from '@views/Home.vue';
+import { Authorize, UnregisteredUser, RegisteredUser, Admin, Moderator, TextManager } from '@views';
 
 Vue.use(VueRouter);
 
 const routes = [
   {
     path: '/',
-    name: 'Home',
-    component: Home
+    name: 'UnregisteredUser',
+    component: UnregisteredUser
+  },
+  {
+    path: '/authorize',
+    name: 'Authorize',
+    component: Authorize
+  },
+  {
+    path: '/registeredUser',
+    name: 'RegisteredUser',
```

Comment:
I don't sure that now we need separate view for unauthorized user. we can delete it now and return to this question after authorization implementation